src/feed/Feed.js

```diff
@@ -272,6 +275,12 @@ class Feed {
         }
     }
 
+    setHasReachedEndOfData(hasReachedEndOfData) {
```

Comment:
@nazaninreihani You can use `Feed.Event_ON_PAGINATION` emitter to set `hasReachedEndOfData` in the corresponding store(`stateStore`).

Reply:
We need to set hasReachedEndOfData to false when starting to fetch a new feed, we can't do that in Feed.Event_ON_PAGINATION, If I want to this I should add an event for starting new feed, what do you think?